code/modules/mob/living/simple_animal/abnormality/waw/contract.dm

```diff
@@ -25,22 +25,55 @@
 		)
 //	gift_type = /datum/ego_gifts/eight
 
+	var/list/total_havers = list()
 
+	var/list/fort_havers = list()
+	var/list/prud_havers = list()
+	var/list/temp_havers = list()
+	var/list/just_havers = list()
 	var/list/spawnables = list()
+	var/total_per_contract = 3
 
 /mob/living/simple_animal/hostile/abnormality/contract/Initialize()
 	. = ..()
-	//We need a list of all abnormalities that are TETH and HE level and Can breach.
-
+	//We need a list of all abnormalities that are TETH to HE level and Can breach.
 	var/list/queue = subtypesof(/mob/living/simple_animal/hostile/abnormality)
 	for(var/i in queue)
 		var/mob/living/simple_animal/hostile/abnormality/abno = i
 		if(!(initial(abno.can_spawn)) || !(initial(abno.can_breach)))
 			continue
 
-		if((initial(abno.threat_level)) <= HE_LEVEL)
+		if((initial(abno.threat_level)) <= WAW_LEVEL)
```

Comment:
Im probably gonna have to retool SoW for this. Probably.

Reply:
Want me to wait?